built_value/lib/built_value.dart

```diff
@@ -95,7 +95,13 @@ class BuiltValueField {
   /// comparing with `operator==`.
   final bool compare;
 
-  const BuiltValueField({this.compare: true});
+  /// Whether the field is serialized. Defaults to `true`.
+  ///
+  /// If a field is not serialized, it must either be `@nullable` or specify a
+  /// default for deserialization to succeed.
```

Comment:
specify a default value how?

Reply:
Currently I believe this has to be by hand-coding the builder.

Reply:
ok, thanks for the feedback